src/components/select/select.jsx

```diff
@@ -177,6 +177,7 @@ const Select = forwardRef(function Select(props, ref) {
         onChange={onSearchChange}
         onClick={onClick}
         onKeyDown={onKeyDown}
+        onKeyUp={props.onKeyUp}
```

Comment:
I guess it's because we want to adhere to an allow-list, but this is part of why I think I would generally just spread remaining props on the native element, we're going to keep finding edge cases where a native element supports something that our wrapper doesn't.

Reply:
Having to create an MDS PR anytime you want to add something to the native element, is enough of a hassle to make someone think: 'ya, I don't want to use this'. We should probably reconsider this at a high level.

Reply:
We moved away from spreading the props because it would lead to usages of components that we are not intending to support. It makes it harder to stay agile when we need to consider esoteric usages in bigmaven.

Reply:
Yeah, I think our reasoning for moving away from this still applies. We want to provide a focused interface to interact with with these components, reduce cognitive load. And even if you do still need to build your own to support some attribute we don't normally (less applicable with something as complex as `Select`, but still), you can still compose in our styles.